fastify.js

```diff
@@ -224,6 +226,17 @@ function build (options) {
     }
 
     req.id = genReqId(req)
+
+    if (trustProxy) {
```

Comment:
the `trustProxy` varialbe should be a boolean check, something like `hasTrustProxy`.